src/legacy/core_plugins/data/public/actions/filters/create_filters_from_event.ts

```diff
@@ -59,22 +71,27 @@ const getOtherBucketFilterTerms = (table, columnIndex, rowIndex) => {
  * Assembles the filters needed to apply filtering against a specific cell value, while accounting
  * for cases like if the value is a terms agg in an `__other__` or `__missing__` bucket.
  *
- * @param  {object} table - tabified table data
+ * @param  {EventData['table']} table - tabified table data
  * @param  {number} columnIndex - current column index
  * @param  {number} rowIndex - current row index
  * @param  {string} cellValue - value of the current cell
- * @return {array|string} - filter or list of filters to provide to queryFilter.addFilters()
+ * @return {Filter[]|undefined} - list of filters to provide to queryFilter.addFilters()
```

Comment:
nit: if its easy to refactor, returning empty array instead of undefined might be easier to consume.